cpp/recognition/CMakeLists.txt

```diff
@@ -4,4 +4,7 @@ koala_add_module(recognition
    perfect/Jewels.cpp
    perfect/Pyramids.cpp
    perfect/NearCleaners.cpp
+
+   planar/IPlanarGraphRecognition.cpp

```

Comment:
Please imitate PerfectGraphRecognition structure, that is:
- move `planar/IPlanarGraphRecognition.cpp` -> `PlanarGraphRecognition.cpp`
- consider moving `planar/HopcroftTarjan.cpp` -> `HopcroftTarjanPlanarGraphRecognition.cpp`
- you don't need an empty line between